dbms/src/Storages/DeltaMerge/SkippableBlockInputStream.h

```diff
@@ -152,6 +152,7 @@ class ConcatSkippableBlockInputStream : public SkippableBlockInputStream
     {
         Block res;
 
+        auto lac_bytes_collector = std::make_unique<LACBytesCollector>(scan_context->resource_group_name);
```

Comment:
How about being member variable of `SkippableBlockInputStream`?

Reply:
If being member of `SkippableBlockInputStream`, the last `LACBytesCollector::consume()` will be called when `SkippableBlockInputStream` is destructed.  Maybe it's too late?

Reply:
done